src/bitops.c

```diff
@@ -16,7 +16,41 @@
 /* Count number of bits set in the binary array pointed by 's' and long
  * 'count' bytes. The implementation of this function is required to
  * work with an input string length up to 512 MB or more (server.proto_max_bulk_len) */
-long long redisPopcount(void *s, long count) {
+#if defined(__GNUC__) || defined(__clang__)
```

Comment:
what about replacing it by using `#if defined(__x86_64__)`?

Reply:
sure